lib/ansible/modules/cloud/docker/docker_swarm_service.py

```diff
@@ -143,13 +144,19 @@
     required: false
     default: 0
     description:
-    - Service memory limit in MB. 0 equals no limit.
+    - "Service memory limit (format: C(<number>[<unit>])). Number is a positive integer.
+      Unit can be C(b) (byte), C(k) (kilobytes, 1024b), C(m) (megabytes), C(g) (gigabyte)."
```

Comment:
```suggestion
      Unit can be C(b) (byte), C(k) (kibibytes, 1024b), C(m) (mebibytes), C(g) (gibibyte)."
```
Sorry I missed that; these are the correct power-of-two units; see https://en.wikipedia.org/wiki/Mebibyte